rotary_bed_MO.py

```diff
@@ -0,0 +1,252 @@
+import numpy as np
```

Comment:
Practically all comments on `kinetics_MO.py` also apply to this example.

Reply:
updated both files.